pom.xml

```diff
@@ -74,13 +74,12 @@
         <bouncy.version>1.55</bouncy.version>
         <xmlsec.version>2.0.8</xmlsec.version>
         <jts.bundle.version>1.12_1</jts.bundle.version>
-        <karaf.version>4.0.7</karaf.version>
+        <karaf.version>4.1.1</karaf.version>
```

Comment:
Do we also need to change the karaf version here: https://github.com/codice/ddf/blob/6fd75e2fdd9a2825002334f5302e1eb8fa2b3249/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L104